e2e/portalicious/tests/RegistrationPage/ViewPersonalInformation.spec.ts

```diff
@@ -49,7 +49,7 @@ test('[34688] User can view the registration data of registration that has all d
 
   const expectedRegistrationData = {
     Name: 'John Snow',
-    FSP: 'ironBank',
+    FSP: 'Iron Bank',
```

Comment:
In the other PRs test-data, the name `ironBank` is still used....
Shouldn't that all be aligned?

https://github.com/global-121/121-platform/pull/6707/files#diff-f77a533d65821f976db28b1f2f7e5bd9b170627d38be28fdf78239980f2b0217R2

Or is this change to _not_ use the 'technical identifier', but the 'human readable string' only?